src/components/IdentityBadge/LocalIdentityBadge.js

```diff
@@ -56,7 +56,7 @@ const LocalIdentityBadge = ({ entity, ...props }) => {
     return () => {
       subscription.unsubscribe()
     }
-  }, [])
+  }, [entity])
```

Comment:
For good habit, we should also put the other dependencies (e.g. `identityEvents$`) in this list